types/polygon-selfintersect/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for polygon-selfintersect 1.0
+// Project: https://github.com/kmcs/polygon-selfintersect
+// Definitions by: Russ McKay <https://github.com/mcruss>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Find self-intersections of the polygon.
+ *
+ * @param points Array of the [x, y] points making up the polygon.
+ * @param findAllLines Find all the intersection lines.
+ * @return Returns `true` if a self-intersection has been found.
+ */
+export function findSelfIntersections(points: readonly number[][], findAllLines?: boolean): boolean;
+
+/**
+ * Get the self-intersection lines.
+ */
+export function getSelfIntersectLines(): number[][];
+
+export as namespace polygonSelfIntersect;
```

Comment:
```suggestion
```
I don't think this was published as UMD module:
https://www.typescriptlang.org/docs/handbook/2/modules.html#umd
https://www.typescriptlang.org/docs/handbook/modules.html#umd-modules